rex/sam_resource.py

```diff
@@ -140,7 +146,11 @@ def __getitem__(self, keys):
             out = self.meta
             out = out.loc[var_slice[0]]
         elif isinstance(var, str):
-            out = self._get_var_ts(var, *var_slice)
+            if var.startswith('mean_'):
```

Comment:
We currently have this linked up in reV to look for "ws_mean", "dni_mean", "ghi_mean", etc...

Reply:
I map those to mean_windspeed, mean_ghi, and mean_dni. We could do var_mean, but ws is not a valid variable in SAMResource

Reply:
Gotcha, I didn't see that, is that in this PR or in reV?

Reply:
reV PR, I just submitted it, FYI the checks will fail